src/components/base/outline-google-map/outline-google-map.ts

```diff
@@ -0,0 +1,248 @@
+import { LitElement, html, CSSResultGroup } from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+import { GoogleMapsApi } from './src/google-maps-api';
+import './src/google-maps-api';
+import { OutlineMapMarker } from './src/outline-map-marker';
+import { MapSelector } from './src/map-selector';
+import './src/map-selector';
```

Comment:
Minor question:

Are there ways to combine these imports? A reason to keep them separate? I'm still trying to understand best practices for imports and the trade-offs.

Reply:
The code works with just importing the entire file, but tsc complains about using the classes OutlineMapMarker and MapSelector without importing it specifically.

$yarn tsc
yarn run v1.22.10

$ /Users/peterschuelke/sites/app.ochsner.org/design/node_modules/.bin/tsc

src/components/base/outline-google-map/outline-google-map.ts:107:13 - error TS2304: Cannot find name 'MapSelector'.

107   _markers: MapSelector;

                ~~~~~~~~~~~

src/components/base/outline-google-map/outline-google-map.ts:107:13 - error TS4031: Public property '_markers' of exported class has or is using private name 'MapSelector'.

107   _markers: MapSelector;

                ~~~~~~~~~~~

Found 2 errors.

error Command failed with exit code 1.

info Visit https://yarnpkg.com/en/docs/cli/run  for documentation about this command.

Reply:
Thanks for the context. :)